### PR TITLE
[SocialMediaBundle] Fix social share bug

### LINKED ISSUE #1193: [SocialMediaBundle] Bug - Partage d'une ressource
| Q             | A
| ------------- | ---
| Bug report?   | yes
| Version?      | master

tentative de partager une ressource (sous menu, option partager) engendre une erreur serveur 

Failed to load resource: the server responded with a status of 500 (Internal Server Error)
http://claroline-master.travis-build.claroline.net/resource/custom/share_action/265

### PR DESCRIPTION
**Pull request by ptsavdar**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Fixed tickets | fixes #1193 

Facebook changed its api format and URL. It needs a token from now on. My code is now fixed for the new "error" format.

### CODE REVIEW

**Comment by purplefish32** [comment]
Funky but LGTM

File: plugin/social-media/Library/SocialShare/Networks/Facebook.php [hunk 1]
```diff
@@ -54,7 +52,7 @@ public function countShares($url)
         $api_url = sprintf(self::API_URL, urlencode($url));
         $data = null;
         try {
-            $data = json_decode(file_get_contents($api_url));
+            $data = json_decode(json_encode(file_get_contents($api_url), true));
```
Comment (stefk) [inline plugin/social-media/Library/SocialShare/Networks/Facebook.php]: Shouldn't the second parameter of `json_encode` be a JSON_* constant bitmask? What's its meaning here?

Comment (ptsavdar) [inline plugin/social-media/Library/SocialShare/Networks/Facebook.php]: Well spotted @stefk ! You're right I wanted to pass the true parameter in json_decode and delete the json_encode. I guess I got lost somewhere between undos and redos :smile:

**Comment by stefk** [comment]
Except for travis, LGTM

**Comment by ptsavdar** [comment]
LGTM

**Pull request merged**